themes/nekolight/nekolight.theme.sh

```diff
@@ -0,0 +1,29 @@
+#! bash oh-my-bash.module
+
+#! bash oh-my-bash.module
+
+_omb_theme_nekolight_version='1.0.0'
+_omb_theme_nekolight_symbol=""
+
+function _omb_theme_nekolight_git_info() {
+  if _omb_prompt_git rev-parse --is-inside-work-tree &>/dev/null; then
+    local branch=$(_omb_prompt_git symbolic-ref --short HEAD 2>/dev/null || _omb_prompt_git rev-parse --short HEAD 2>/dev/null)
+
+    if _omb_prompt_git diff --quiet 2>/dev/null && _omb_prompt_git diff --cached --quiet 2>/dev/null; then
+      echo "${_omb_prompt_bold_green}${branch} ${_omb_theme_nekolight_symbol}${_omb_prompt_normal}"
+    else
+      echo "${_omb_prompt_bold_red}${branch} ${_omb_theme_nekolight_symbol}${_omb_prompt_normal}"
+    fi
+  else
+    echo ""
```

Comment:
This doesn't have any effect because `$()` will anyway remove the newlines at the end of the result.

Reply:
Thanks!